tailwind.config.js

```diff
@@ -191,6 +191,14 @@ module.exports = {
         'xxs': ['0.6rem', '0.6rem']
       },
     },
+    listStyleType: {
```

Comment:
what is this one used for?

Reply:
I used this is to change the list style from `dot/circle` to `square` when there is a need for nested bullet-points. I used this in `press` page but it is not used now (as that page is removed). If you want, we can remove these lines.

Reply:
yes, please remove